apps/workers/sql-load/README.md

```diff
@@ -39,5 +39,37 @@ Example tasks:
 
 ## Technical: 
 
+### JSON and ARRAY postgres types
+When loading json or jsonb column types you can load the following examples
+- `["1","2","3"]`
+- `{"column_name":{"field":"value"}}`
+
+When loading ARRAY postgres types, if the value is a json object array it can be loaded normally.
+- `[1,"two","three"]`
+
+When loading a quoted string for an ARRAY type it must be enclosed in `{}` curly braces
+- `"{1,2,3}"`
+
+### Below are examples of data loading for CSV and JSON files
+
+#### CSV files and loading JSON and ARRAY postgres types
+When loading a csv file that has json types values, or array types.
+You must double the double quotes to escape the double quotes correctly.
+Here is an example of a properly formatted csv file with json and array types.
```

Comment:
When loading a csv file that has json or array types, 
you must escape quotes be double quoting them. See below: